### PR TITLE
docs: Adds ADR 10. Localization

### PR DESCRIPTION
**Pull request by pomegranited**
### Description

Documents our decision to translate Superset assets.

View rendered page here: [10. Localization of user-facing content](https://docsopenedxorg--107.org.readthedocs.build/projects/openedx-aspects/en/107/decisions/0010_localization.html)

### Supporting information

Part of https://github.com/openedx/openedx-aspects/issues/61

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: docs/decisions/0010_localization.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  logging in; this will not be chosen automatically from their Open edX preferred language.
+* Aspects will contribute to Superset's i18n and l10n efforts where possible.
+
+Consequences
+************
+
+#. Aspects will configure Superset to run with a set of locales that best matches those locales Open edX supports.
+#. Aspects will create translated copies of all user-facing Superset assets (dashboards and charts).
+#. Like other Open edX projects, translations for the Aspects project will be maintained in Transifex.
+#. Translations will be automatically synchronized using the Transifex Github App as per `OEP-58`_.
```
Comment (Ian2012) [inline docs/decisions/0010_localization.rst:52]: Please add information on how Dashboard RBAC is used to assign dashboards to users, and how the base roles are used too.

**Review by Ian2012 — CHANGES_REQUESTED** [review]

File: docs/decisions/0010_localization.rst (hunk 1 above)
Comment (pomegranited) [reply docs/decisions/0010_localization.rst:55]: How's this? [df0b52c](https://github.com/openedx/openedx-aspects/pull/107/commits/df0b52c4bd60a7ed2859582aec5e9e3ab744b079)

File: docs/decisions/0010_localization.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+#. Access to these translated dashboards will be gated by role-based access controls (RBAC) using locale-specific roles.
+   Users are added to the locale-specific role that best matches their preferred language in Open edX, and so are
+   granted access to the appropriate translated dashboards and charts.
+#. Like other Open edX projects, translations for the Aspects project will be maintained in Transifex.
+#. Translations will be automatically synchronized using the Transifex Github App as per `OEP-58`_.
+
+Rejected Alternatives
+*********************
+
+**Translate assets on the fly with javascript**
```
Comment (pomegranited) [inline docs/decisions/0010_localization.rst:60]: @Ian2012 Are there any other rejected alternatives that I should document here? I just made this one up..

File: docs/decisions/0010_localization.rst (hunk 1 above)
Comment (Ian2012) [reply docs/decisions/0010_localization.rst:55]: yes, thanks

**Review by Ian2012 — APPROVED** [review]

File: docs/decisions/0010_localization.rst (hunk 2 above)
Comment (Ian2012) [reply docs/decisions/0010_localization.rst:60]: Give support for asset translations in Superset https://github.com/apache/superset/issues/13442

**Review by bmtcril — APPROVED** [review]

File: docs/decisions/0010_localization.rst (hunk 2 above)
Comment (pomegranited) [reply docs/decisions/0010_localization.rst:60]: Addressed with https://github.com/openedx/openedx-aspects/pull/110

**Pull request merged**